modules/mutect2/2.0/mutect2.smk

```diff
@@ -47,6 +47,12 @@ rule _mutect2_input_bam:
         op.relative_symlink(input.bam, output.bam)
```

Comment:
Does this work on cram files as-is? If not, could you please add the second crai symlink with the ".crai" name so it can be automatically found?

Reply:
Good question! I will actually test this on the Kridel CRAM files before I merge this.

Reply:
No .crai symlink is needed, but I added the reference fasta to one of the other Mutect rules and it runs perfectly on the Kridel CRAM files now.

Reply:
Awesome